libraries/commerce/favorites/actions/fetchFavorites.js

```diff
@@ -21,12 +22,12 @@ import {
 const fetchFavorites = (ignoreCache = false) => (dispatch, getState) => {
   const data = getState().favorites.products;
   if (!ignoreCache && !shouldFetchData(data)) {
-    return new Promise(res => res());
+    return Promise.resolve();
   }
   const timestamp = Date.now();
   dispatch(requestFavorites());
   const promise = new PipelineRequest(pipelines.SHOPGATE_USER_GET_FAVORITES)
-    .setErrorBlacklist([EFAVORITE, EUNKNOWN, EBIGAPI])
+    .setErrorBlacklist([EFAVORITE, EUNKNOWN, EBIGAPI, ELIMIT])
```

Comment:
Why does `ELIMIT` have to be blacklisted for the `get favorites` pipeline call? The pipeline doesn't look like it's even returning this error.

Reply:
Since it does no harm, it can stay like this for now.